benchmarks/holoscan_flow_benchmarking/benchmark.py

```diff
@@ -272,19 +275,24 @@ def main():
                 stop_gpu_monitoring_lock.release()
                 gpu_monitoring_thread.join()
                 gpu_utilization_log_files.append(gpu_utilization_logfile_name)
-            print(f"Run {i} completed for {scheduler} scheduler.")
+            logging.info(f"Run {i} completed for {scheduler} scheduler.")
             time.sleep(1)  # cool down period
 
+    def print_missing(files: list) -> None:
+        absolute_filepaths = [os.path.abspath(os.path.join(log_directory, file)) for file in files]
+        missing_files = [file for file in absolute_filepaths if not os.path.exists(file)]
+        if missing_files:
+            logging.error(f"Log file(s) {', '.join(missing_files)} are missing.")
```

Comment:
```suggestion
    def print_missing(files: list) -> bool:
        absolute_filepaths = [os.path.abspath(os.path.join(log_directory, file)) for file in files]
        missing_files = [file for file in absolute_filepaths if not os.path.exists(file)]
        if missing_files:
            logging.error(f"Log file(s) {', '.join(missing_files)} are missing.")
            return true
        return false  
```

Something like the above could be done.

Reply:
Thanks for the suggestion, I agree with the goal here, it is useful to have missing/present file results available for more granular consideration. Removed this function in favor of an inlined block.